README.md

```diff
@@ -174,12 +174,27 @@ sudo make install
 ```
 
 # Test
-the e2fsprogs live in /opt/gnu/bin and /opt/gnu/sbin, fuse-ext2 is in /usr/local/bin
+
+The e2fsprogs live in /opt/gnu/bin and /opt/gnu/sbin, fuse-ext2 is in /usr/local/bin
+
+```shell
+cd
+dd if=/dev/zero of=/tmp/test-fs.ext4 bs=1024 count=102400  
+/opt/gnu/sbin/mkfs.ext4 /tmp/test-fs.ext4
+mkdir -p ~/mnt/fuse-ext2.test-fs.ext4
+fuse-ext2  /tmp/fuse-ext2.test-fs.ext4 -o rw+,allow_other,uid=501,gid=20
```

Comment:
This needs the device and mount point corrected on the fuse-ext2 line:

     fuse-ext2  /tmp/test-fs.ext4 mnt/fuse-ext2.test-fs.ext4 -o rw+,allow_other,uid=501,gid=20